### ISSUE TITLE
JSON serialization for Models items() conflict

### ISSUE THREAD

**Issue by APinchofDill** [AtomHash/evernode#37]
**Summary**
If an attribute is added to the Models called "items" it will override items() on the vars() dict. Thus the Json class cannot iterate over the values and causes a 502 http soft error.

**Steps to reproduce issue**
1. Create a model that inherits from BaseModel. 
2. Dynamically add an attribute called "items"`model.add('items', [])`
3. Try to do a HTTP response

**Expected results**
serialize and return HTTP response.

**Actual results**
causes a 502 HTTP error. The error: `'list' object is not callable, json.class line 117`

**Comment by APinchofDill** [comment]
**Suggested fix**
```
for key in obj:
    value = obj[key]
```

**Comment by APinchofDill** [comment]
In terms of performance use:
```
if hasattr(obj, 'items') and \
                isinstance(obj.items, types.BuiltinFunctionType):
```
and keep .items() iterator.

**Comment by APinchofDill** [comment]
Fixed 5be4970b902ab5facc9d7417ffd9342899eacb00
